packages/shared/components/Text.svelte

```diff
@@ -45,7 +45,7 @@
         }
     }
     pre {
```

Comment:
we could increase the `pre` font size to 12px `text-12` to match the figma designs